install/resources/mybb_theme.xml

```diff
@@ -3966,14 +3966,14 @@ var announcement_quickdelete_confirm = "{$lang->announcement_quickdelete_confirm
 {$forums}
 </table>
 <br />]]></template>
-		<template name="forumdisplay_thread" version="1813"><![CDATA[<tr class="inline_row">
+		<template name="forumdisplay_thread" version="1823"><![CDATA[<tr class="inline_row">
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%"><span class="thread_status {$folder}" title="{$folder_label}">&nbsp;</span></td>
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%">{$icon}</td>
 	<td class="{$bgcolor}{$thread_type_class}">
 		{$attachment_count}
 		<div>
           <span>{$prefix} {$gotounread}{$thread['threadprefix']}<span class="{$inline_edit_class} {$new_class}" id="tid_{$inline_edit_tid}"><a href="{$thread['threadlink']}">{$thread['subject']}</a></span>{$thread['multipage']}</span>
-			<div class="author smalltext">{$thread['profilelink']}</div>
+			<div class="author smalltext">{$thread['profilelink']} <span class="smalltext" title="{$lang->thread_started_time_date} {$threaddate}, {$threadtime}">({$threaddate}, {$threadtime})</span></div>
```

Comment:
I wonder if it would be best to have language a variable for the string, and to use `$thread['date']` and `$thread['time']` for the variables, then do `$thread_date_and_time_title = $lang->sprintf($lang->'thread_started_time_date, $thread['date'], $thread['time'])` and `$thread_date_and_time = $lang->sprintf($lang->'thread_started_time_date_value, $thread['date'], $thread['time'])`?

This way, the `,` isn't hard-coded as I don't know if that punctuation makes sense for all languages?

Reply:
@euantorano - please check now